modules/040-control-plane-manager/docs/FAQ_RU.md

```diff
@@ -280,10 +280,12 @@ for pod in $(kubectl get pod -n kube-system -l component=etcd,tier=control-plane
 done
 ```
 
-В текущей директории будет создан файл `etc-backup.snapshot` со снимком базы etcd одного из членов etcd-кластера.
+В текущей директории будет создан файл `etcd-backup.snapshot` со снимком базы etcd одного из членов etcd-кластера.
 Из полученного снимка можно будет восстановить состояние кластера etcd.
 
-О возможных вариантах восстановления состояния кластера etcd из снимка вы можете узнать [здесь](https://github.com/deckhouse/deckhouse/blob/main/modules/040-control-plane-manager/docs/internal/ETCD_RECOVERY.md).
+Также рекомендуем сделать бекап директории `/etc/kubernetes/pki/etcd` в которой находятся [ключи и сертификаты доступа](https://etcd.io/docs/v3.5/op-guide/security/) к etcd-серверу.  
 
-Мы рекомендуем хранить резервные копии снимков состояния кластера etcd в зашифрованном виде вне кластера Deckhouse.
+Мы рекомендуем хранить резервные копии снимков состояния кластера etcd, а также ключи и сертификаты доступа к etcd-сервера в зашифрованном виде вне кластера Deckhouse.
```

Comment:
```suggestion
Мы рекомендуем хранить резервные копии снимков состояния кластера etcd, а также ключи и сертификаты доступа к etcd-серверу в зашифрованном виде вне кластера Deckhouse.
```

Reply:
I've fixed `etcd-сервера` but i do not agree with Deckhouse -> Kubernetes

Reply:
I suggest the change because we haven't used such a term before - 'Deckhouse cluster'. Maybe something like "... outside the cluster with the Deckhouse"?

Reply:
I found multiple usages of "Deckhouse cluster" in our documentation. [Check it!](https://www.google.com/search?q=site%3Adeckhouse.io+%22Deckhouse+cluster%22)

Reply:
I think, main goal of Deckhouse it is hiding Kubernets behind itself, and we can using "Deckhouse cluster" as normal term (or phrase).

Reply:
ok